installation/standard.py

```diff
@@ -48,8 +48,8 @@
 #     If you set x11libdir (that is two ones and an el) AND x11include to point
 #     to the lib and include directories, they will be used.
 #     Otherwise a search is made for common locations.
-x11libdir=''
-x11include=''
+x11libdir='/usr/X11R6/lib'
+x11include='/usr/X11R6/include'
```

Comment:
I don't think there is a system libpng besides the xquartz one at least.  If we detect that and compile our own libpng and the user later installs xquartz, there is a good chance it will segfault as it did before we started using the xquartz version a while back.

XQuartz is so standard these days most everybody installs it even if they don't know it.  There is even a GUI dialog for installing it that you can trigger from the commandline.